api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -28,5 +28,8 @@ private FhirConstants() {
 	public static String OPENMRS_FHIR_EXT_NAME = OPENMRS_FHIR_EXT_PREFIX + "/name";
 	
 	public static String PATIENT = "Patient";
+
+	@Value("${project.parent.artifactId}.personAttributeTypeUuid")
+	public static String PERSON_ATTRIBUTE_TYPE_PROPERTY;
```

Comment:
How about renaming this class to some thing like `FhirProperties`. Otherwise, looking at constants which can be modified (are not final) makes me think otherwise.